### ISSUE TITLE
default config does not forward tcp/80,443

### ISSUE THREAD

**Issue by cameroncking** [iflowfor8hours/sandcastle#24]
Under the default config, vagrant will not forward tcp/80 if the user is logged in as a non-root user.  Users should be encouraged to run services as non-root users.  This leads to a discrepancy between the default config and the instructions in the README.

Specifically, when users are directed to open https://sandstorm.io.vagrant.dev, they will get "server unavailable".

**Comment by cameroncking** [comment]
There are a number of ways to address this issue:
1. Update README to instruct users to add `172.19.22.22 sandstorm.io.vagrant.dev` to their /etc/hosts file.
2. Use "sandstorm.172.19.22.22.xip.io" as the hostname instead of "sandstorm.io.vagrant.dev".  This adds a dependency on an additional third party, but it does have the benefit of requiring no root changes to the system.  It also supports wildcards.
3. Give the user instructions on how to perform iptables forwarding to an alternative port, such as tcp/8080.  Popular Linux distributions may already have this information in their wikis.
4. Abandon tcp/80,443 all together and add forwarding toa  high-numbered port such as tcp/8080 and update README to direct the user to https://sandstorm.io.vagrant.dev:8080.

I use option 2 in my local Vagrantfile, and it works great.

**Comment by jacksingleton** [comment]
Did you run into this problem with landrush up and running? I would guess this problem was more to do with the DNS config than port binding.

We can't use the hosts method because sandstorm needs a wildcard DNS entry, which isn't possible using /etc/hosts. So we switched to landrush which uses dnsmasq to resolve the dev domain.

But I hadn't heard of xip.io, that service looks awesome! Landrush has caused us some pain, I'd be down with ripping it out of the setup instructions and just referencing `sandstorm.172.19.22.22.xip.io`

Can you confirm if the problems are DNS or port related?

**Comment by cameroncking** [comment]
In my cause, my user id (uid=1000) in a default Ubuntu install does not have sufficient privileges to open tcp/80 or 443.  Therefore, vagrant does not forward the port.  Since "sandstorm.io.vagrant.dev" resolves to the loopback, I was unable to connect over http.  Changing the port to tcp/8080 worked fine.

I did install landrush, but I did not complete the dnsmasq setup.  That was probably my issue.
